graphql/src/schema/api.rs

```diff
@@ -417,30 +497,53 @@ fn add_subscription_type(
     Ok(())
 }
 
+fn block_argument() -> InputValue {
+    InputValue {
+        position: Pos::default(),
+        description: None, /*Some(
+                               "The block at which the query should be executed. \
+                                Can either be an `Int` containing the block number or a `Bytes` \
+                                value containing a block hash. Defaults to the latest block when \
+                                omitted."
+                                   .to_owned(),
+                           ), */
```

Comment:
Is there any reason for this to be commented out?

Reply:
Nope, just an oversight from testing. Putting it back in